src/Native/Unix/System.Security.Cryptography.Native/pal_pkcs7.c

```diff
@@ -24,22 +24,9 @@ PKCS7* CryptoNative_D2IPkcs7Bio(BIO* bp)
     return d2i_PKCS7_bio(bp, NULL);
 }
 
-PKCS7* CryptoNative_Pkcs7CreateSigned()
+PKCS7* CryptoNative_Pkcs7Sign(X509Stack* certs)
```

Comment:
If it's "Pkcs7Sign" it should give all the parameters that the wrapped function does (this version doesn't actually sign anything).

Alternatively, changing the name to something like "Pkcs7CreateCertificateCollection" would make it more clear why it doesn't take any other parameters.